packages/server/postgres/queries/getTeamPromptResponsesByIds.ts

```diff
@@ -1,21 +1,41 @@
 import {JSONContent} from '@tiptap/core'
+import TeamPromptResponseId from 'parabol-client/shared/gqlIds/TeamPromptResponseId'
+import {MaybeReadonly} from 'parabol-client/types/generics'
+import Reactji from '../../database/types/Reactji'
 import getPg from '../getPg'
 import {
   getTeamPromptResponsesByIdsQuery,
   IGetTeamPromptResponsesByIdsQueryResult
 } from './generated/getTeamPromptResponsesByIdsQuery'
 
 export interface TeamPromptResponse
-  extends Omit<IGetTeamPromptResponsesByIdsQueryResult, 'content'> {
+  extends Omit<IGetTeamPromptResponsesByIdsQueryResult, 'content' | 'reactjis' | 'id'> {
+  id: string
+  reactjis: Reactji[]
   content: JSONContent
 }
 
+export const mapToTeamPromptResponse = (
+  results: IGetTeamPromptResponsesByIdsQueryResult[]
+): TeamPromptResponse[] => {
+  return results.map((teamPromptResponse: any) => {
+    return {
+      ...teamPromptResponse,
+      id: TeamPromptResponseId.join(teamPromptResponse.id),
+      reactjis: teamPromptResponse.reactjis.map(
+        (reactji: {shortname: string; userid: string}) =>
+          new Reactji({id: reactji.shortname, userId: reactji.userid})
```

Comment:
+1 would be nice to use the same structure `{id, userId}` in Postgres so we could get away with just casting.

Reply:
Could you elaborate what you mean by SQL's case-insensitivity? The columns have the exact case as defined.